docs/program/apis/_index.md

```diff
@@ -332,105 +276,71 @@ Documentation on using these methods in your SDK code is found on [service pages
 
 ### Motion
 
-Method Name | Description
------------ | -----------
-[`Move`](/services/motion/#move) | Move multiple components in a coordinated way to achieve a desired motion.
-[`MoveSingleComponent`](/services/motion/#movesinglecomponent) | Move a single component "manually."
-[`GetPose`](/services/motion/#getpose) | Get the current location and orientation of a component.
-[`MoveOnMap`](/services/motion/#moveonmap) | Move a [base](/components/base/) component to a `Pose` in respect to the origin of a [SLAM](/services/slam/) map.
-[`MoveOnGlobe`](/services/motion/#moveonglobe) | Move a [base](/components/base/) component to a destination GPS point. Use a [Movement Sensor](/components/movement-sensor/) to measure the robot's GPS coordinates.
+The [Motion Service](/services/motion/) supports the following methods:
 
-### SLAM
+{{< readfile "/static/include/services/apis/motion.md" >}}
 
-Method Name | Description
------------ | -----------
-[`GetPosition`](/services/slam/#getposition) | Get the current position of the specified source component in the point cloud SLAM map.
-[`GetPointCloudMap`](/services/slam/#getpointcloudmap) | Get the point cloud SLAM map.
-[`GetInternalState`](/services/slam/#getinternalstate) | Get the internal state of the SLAM algorithm required to continue mapping/localization.
-[`GetLatestMapInfo`](/services/slam/#getlatestmapinfo) | Get the timestamp of the last update to the point cloud SLAM map.
+### Navigation
 
-### MLModel
+The [Navigation Service](/services/navigation/) supports the following methods:
+
+{{< readfile "/static/include/services/apis/navigation.md" >}}
+
+### Sensors
 
-Method Name | Description
------------ | -----------
-`Infer` | Take an already ordered input tensor as an array, make an inference on the model, and return an output tensor map.
-`Metadata`| Get the metadata (such as name, type, expected tensor/array shape, inputs, and outputs) associated with the ML model.
+The [Sensors Service](/services/sensors/) supports the following methods:
+
+{{< readfile "/static/include/services/apis/sensors.md" >}}
+
+### SLAM
+
+The [SLAM Service](/services/slam/) supports the following methods:
+
+{{< readfile "/static/include/services/apis/slam.md" >}}
 
 ### Vision
 
-Method Name | Description |
------------ | ----------- |
-`DetectionsFromCamera` | Get a list of detections in the next image given a camera and a detector. |
-`Detections`| Get a list of detections in the given image using the specified detector. |
-`ClassificationsFromCamera` | Get a list of classifications in the next image given a camera and a classifier. |
-`Classifications` | Get a list of detections in the given image using the specified detector. |
-`ObjectPointClouds`| Returns a list of the 3D point cloud objects and associated metadata in the latest picture obtained from the specified 3D camera (using the specified segmenter). |
+Different [Vision Service](/services/vision/) models support different methods:
```

Comment:
```suggestion
Different models of the [Vision Service](/services/vision/) support different methods:
```
or 
```suggestion
Different API methods are available for each model of [Vision Service](/services/vision/):
``` 
/ endpoints 

(opt. for user comprehension!)

Reply:
Mm I think since the Vision Service is one thing, the current wording makes the most sense? I might be missing what you're trying to clarify though

Reply:
I think any of these three work 🤷‍♀️